stingray/deadtime/fad.py

```diff
@@ -195,15 +215,46 @@ def calculate_FAD_correction(lc1, lc2, segment_size, gti=None,
         c = (f2 * f1.conj()).real
         c = c / smooth_real * 2
 
+        if norm == "none":
```

Comment:
I would suggest moving this whole `if-then-else` clause to a dedicated function for the normalization of power spectra. And maybe use the same function here and in the methods of the `*Powerspectrum` classes to avoid code duplication. 
Also, this would allow us testing the normalization function properly, separated from the rest of the machinery

Reply:
Made a new function `normalize_crossspectrum` in `crossspectrum.py` that is basically just the code from the respective method in `Crossspectrum`. That now has its own tests, and both the Fourier classes and the FAD code use that implementation.